testsuite/MDAnalysisTests/analysis/test_hole2.py

```diff
@@ -403,7 +404,11 @@ def test_plot_mean_profile(self, hole, frames, profiles):
         binned, bins = hole.bin_radii(bins=100)
         mean = np.array(list(map(np.mean, binned)))
         stds = np.array(list(map(np.std, binned)))
-        midpoints = 0.5 * bins[1:] + bins[:-1]
+        midpoints = 0.5 * bins[1:] + 0.5 * bins[:-1]
+        # assume a bin of length 1.5
+        difference_right = bins[1:] - midpoints
+        assert_allclose(difference_right, 0.75)
```

Comment:
This one currently fails as you can see eg in https://github.com/MDAnalysis/mdanalysis/runs/6858196294?check_suite_focus=true#step:7:520

```
        difference_right = bins[1:] - midpoints
>       assert_allclose(difference_right, 0.75)
E       AssertionError: 
E       Not equal to tolerance rtol=1e-07, atol=0
E       
E       Mismatched elements: 100 / 100 (100%)
E       Max absolute difference: 0.54100065
E       Max relative difference: 0.7213342
E        x: array([0.208999, 0.208999, 0.208999, 0.208999, 0.208999, 0.208999,
E              0.208999, 0.208999, 0.208999, 0.208999, 0.208999, 0.208999,
E              0.208999, 0.208999, 0.208999, 0.208999, 0.208999, 0.208999,...
E        y: array(0.75)
```

Adding a comment here to document your assumption was a good idea. I don't think you can just assume that the bin size is 1.5. You could calculate the bin width as 
```python
binwidths = np.diff(bins)
binwidth = binwidths[0]
assert_allclose(binwidths, bindwidth)  # just making sure that we have equidistant bins

difference_right = bins[1:] - midpoints
assert_allclose(difference_right, binwidth/2)
```